docs/UserGuide.md

```diff
@@ -116,12 +118,12 @@ Examples:
 
 ### Editing a person: `edit`
 
-Edits the specified person in the person list.
+Edits the specified contact detail in the person list.
```

Comment:
IMO we shouldn't introduce new terms like "contact" here. I'd prefer sticking to the terms "person" and "group" only.